lib/numbers.md

```diff
@@ -5,20 +5,49 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Fixnum : An integer (a whole number with no decimal point).
+Float : A number with a decimal. eg: 3.14
+
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+.abs : Absolute value (numerical value, regardless of whether it is positive or negative.
+% : Modulo, or the remainder of a division operation.
+** : Exponent
+/ : Division
++ : Addition
+- : Subtraction
+* : Multiplication
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+Used with a number, the '+' operation will perform additive math. Used with a String, the '+' operation will merely place the strings next to one another.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, you must convert the string to an integer before performing math operations on it. 
+Use:
+'20'.to_i
```

Comment:
What if you wanted to divide the string `"20"` by an odd number–like 3? What problem would using `to_i` cause, and how would you avoid it?